app/mailers/ticket_mailer.rb

```diff
@@ -124,7 +124,8 @@ def receive(email)
         message_id: email.message_id,
         content_type: content_type,
         to_email_address: to_email_address,
-        raw_message: StringIO.new(email.to_s)
+        raw_message: StringIO.new(email.to_s),
+        unread_users: User.all # none of the users could have read this
```

Comment:
And ran a dirty clean-up hack on the console:

```
class TicketsUser < ApplicationRecord
end

TicketsUser.count # before: 2077962
TicketsUser.delete_all(user_id: User.where(agent: false).pluck(:id))
TicketsUser.count # after: 22683
```

Now load times are again < 1s :)